lib/grape-active_model_serializers/endpoint_extension.rb

```diff
@@ -37,8 +37,15 @@ def serialization_scope
       end
     end
 
-    def render(resources, meta = {})
-      env['ams_meta'] = meta
+    def render(resources, extra_options = {})
+      extra_options.symbolize_keys!
+      env['ams_meta_options'] = extra_options.slice(
```

Comment:
I think just in case someone relied on this I would rename `ams_meta_options` to just `ams_meta` and similar for the others. The `options` part is redundant anyway.